test/poolV2.js

```diff
@@ -173,4 +172,22 @@ contract('Pool', accounts => {
         /* = 1 */
     });
 
+    it('should be able to exercise a Prime', async () => {
+        let tokenId = await _prime.nonce();
+
+        /* Bob then exercises the Prime */
+        let collateralAmount = await web3.utils.toWei('0.1');
+        let strikeAmt = await web3.utils.toWei('1');
+        await _tUSD.approve(_prime.address, strikeAmt, {from: Bob});
+        await _tUSD.mint(Bob, strikeAmt);
+
+        console.log('Exercises ether denominated prime');
```

Comment:
we should probably start deleting console.log statements. They add up and get messy pretty quick. If we want handmade logs, we should implement a more robust logging system like [pino](https://getpino.io/#/).

Reply:
I'll be doing assert statements and no console logs. I was basically acting as a human assert by checking the logs